pages/requests/[id].js

```diff
@@ -37,17 +38,37 @@ const Request = () => {
 
   if (isError) return <Error errors={configureErrors([isRequestError, isSOWError, isMessagesError])} router={router} />
 
-  const handleSendingMessages = ({ message, files }) => {
-    postMessageOrAttachment({ id, message, files })
-    mutate({ ...data, ...messages })
+  if (!session) {
```

Comment:
the "/" and "/browse" routes ought to be accessible at all times. whether with the user's access token or the supplier's. request routes however should always require a user.

at the moment, this is being tracked by checking for a session.